src/main/java/org/akip/web/rest/ProcessMemberController.java

```diff
@@ -0,0 +1,79 @@
+package org.akip.web.rest;
+
+import org.akip.domain.ProcessMember;
+import org.akip.service.ProcessMemberService;
+import org.akip.service.dto.ProcessMemberDTO;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import tech.jhipster.web.util.HeaderUtil;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+
+/**
+ * REST controller for managing {@link ProcessMember}.
+ */
+@RestController
+@RequestMapping("/api")
+public class ProcessMemberController {
+
+    private final Logger log = LoggerFactory.getLogger(ProcessMemberController.class);
+
+    private static final String ENTITY_NAME = "processMember";
+
+    private static final String MESSAGE_PROCESS_MEMBER_UPDATED = "Process Member Successfully Updated";
+
+    private static final String MESSAGE_PROCESS_MEMBER_CREATED = "Process Member Successfully Created";
+
+    private static final String MESSAGE_PROCESS_MEMBER_REMOVED = "Process Member Successfully Removed";
+
+
+    @Value("${jhipster.clientApp.name}")
+    private String applicationName;
+
+    private final ProcessMemberService processMemberService;
+
+    public ProcessMemberController(ProcessMemberService processMemberService) {
+        this.processMemberService = processMemberService;
+    }
+
+    @GetMapping("/process/{processId}/members")
```

Comment:
processId? Do you mean processDefinitionId? If so, you should keep the consistency with the existing controllers and name it processDefinitionId.

```suggestion
    @GetMapping("/process-definition/{processDefinitionId}/members")
```